Documentation/leak-detection.md

```diff
@@ -0,0 +1,24 @@
+# Leak detection (poisoning)
+
+## Overview
+
+Source-build includes a mechanism for *poisoning* its input files, and then for checking for that poison after the build is complete.  This allows us to ensure that all output files were built during the build rather than copied directly from the input, which would be an illegal prebuilt usage.
```

Comment:
```suggestion
Source-build includes a mechanism for *poisoning* its input files, and then for checking for that poison int the build output.  This allows us to ensure that all output files were built during the build rather than copied directly from the input, which would be an illegal prebuilt usage.
```

Reply:
Changed as suggested.